examples/PathChecking.elm

```diff
@@ -7,12 +7,13 @@ import Script.File as File
 
 niceScript : Directory permissions -> Script String ()
 niceScript directory =
-    File.read (File.in_ directory "test.txt")
+    File.read (File.in_ directory "subdirectory/../test.txt")
         |> Script.thenWith
             (\contents ->
                 Script.printLine <|
                     String.fromInt (String.length contents)
-                        ++ " characters in test.txt"
+                        ++ " characters in test.txt\r\n\r\n"
+                        ++ "The following lines should indicate a path access error:\r\n\r\n"
```

Comment:
Is `\r` actually required here? I'd be inclined to just use `\n` for simplicity...

Reply:
Sure, if it works on Windows I'm all for it.

Reply:
Yeah, I think it should be fine - I use Windows almost exclusively and I think the only thing that actually gets confused by a plain `\n` is Notepad =)